pyls/_utils.py

```diff
@@ -140,18 +139,32 @@ def format_docstring(contents):
     """
     contents = contents.replace('\t', u'\u00A0' * 4)
     contents = contents.replace('  ', u'\u00A0' * 2)
-    if LooseVersion(JEDI_VERSION) < LooseVersion('0.15.0'):
-        contents = contents.replace('*', '\\*')
     return contents
 
 
 def clip_column(column, lines, line_number):
-    # Normalise the position as per the LSP that accepts character positions > line length
-    # https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#position
+    """Normalise the position as per the LSP that accepts character positions > line length
```

Comment:
```suggestion
    """
    Normalise the position as per the LSP that accepts character positions > line length
```

Reply:
Very picky considering how the rest of the file looks. PEP 257 allows both, even favors the original one. And don't suggest stuff, if you can't wait and push the commit by yourself a few minutes later. We have been waiting for your review for weeks.